tests/unit_tests/include/test_helper.hpp

```diff
@@ -217,6 +217,7 @@ static inline void aligned_free(void *p) {
 
 /* Support for Unified Shared Memory allocations for different backends */
 static inline void *malloc_shared(size_t align, size_t size, sycl::device dev, sycl::context ctx) {
+    (void)align;
```

Comment:
```suggestion
static inline void *malloc_shared(std::size_t align, std::size_t size, sycl::device dev, sycl::context ctx) {
```

Reply:
I'm not sure about this, most of the common oneMKL code uses `size_t`. I have a preference for `std::size_t` which we use mostly in the DFT domain. I think it's more consistent if we don't change the common code.